frame/support/src/traits/misc.rs

```diff
@@ -1154,6 +1154,18 @@ impl<Hash> PreimageRecipient<Hash> for () {
 	fn unnote_preimage(_: &Hash) {}
 }
 
+/// Trait for creating an asset account with a deposit taken from a specified by client depositor.
+pub trait AccountTouch<AssetId, AccountId> {
+	/// The type for currency units of the deposit.
+	type Balance;
+
+	/// The deposit amount of a native currency required for creating an asset account.
+	fn deposit() -> Self::Balance;
```

Comment:
`deposit` is a verb as well as a noun; as a function you'll tend to read the verb form of a word over the noun-form.